test/test_datasets_samplers.py

```diff
@@ -38,13 +38,13 @@ def get_list_of_videos(num_videos=5, sizes=None, fps=None):
         yield names
 
 
-@unittest.skipIf(not io.video._av_available(), "this test requires av")
-class Tester(unittest.TestCase):
+@pytest.mark.skipif(not io.video._av_available(), reason="this test requires av")
+class TestDatasetsSamplers:
     def test_random_clip_sampler(self):
         with get_list_of_videos(num_videos=3, sizes=[25, 25, 25]) as video_list:
             video_clips = VideoClips(video_list, 5, 5)
             sampler = RandomClipSampler(video_clips, 3)
-            self.assertEqual(len(sampler), 3 * 3)
+            assert_equal(len(sampler), 3 * 3)
```

Comment:
Thanks for the PR @vivekkumar7089 !

Since we're just comparing numbers here (not tensors), we can just rely on a simple form:
```suggestion
            assert len(sampler) == 3 * 3
```

Could you please update this one and all of the remaining ones below as well? Thanks!

Reply:
OK, I will update.

Reply:
@vivekkumar7089 Sorry for the confusion again but that comment above still stands: we should convert `assertEqual` into `assert a == b`, not into `assert_equal`

Reply:
No problem, @NicolasHug , I have changed it back to `assert a == b`.